v5_position_service.go

```diff
@@ -329,3 +330,48 @@ func (s *V5PositionService) GetClosedPnL(param V5GetClosedPnLParam) (*V5GetClose
 
 	return &res, nil
 }
+
+// V5SwitchPositionMarginModeParam :
+type V5SwitchPositionMarginModeParam struct {
+	Category     CategoryV5         `json:"category"`
+	TradeMode    PositionMarginMode `json:"tradeMode"`
+	Symbol       SymbolV5           `json:"symbol"`
+	BuyLeverage  string             `json:"buyLeverage"`
+	SellLeverage string             `json:"sellLeverage"`
+}
+
+func (p V5SwitchPositionMarginModeParam) validate() error {
+	if p.Category != CategoryV5Linear && p.Category != CategoryV5Inverse {
+		return fmt.Errorf("only linear and inverse are supported for category")
+	}
+	if p.Symbol == "" || p.BuyLeverage == "" || p.SellLeverage == "" {
+		return fmt.Errorf("Symbol, BuyLeverage and SellLeverage needed")
+	}
+	return nil
+}
```

Comment:
👍

Reply:
💭 It would be good to check the match between sellLeverage and buyLeverage.

<img width="772" alt="スクリーンショット 2023-05-17 18 15 49" src="https://github.com/hirokisan/bybit/assets/12891407/9507cabf-c53e-4c14-bf81-297e94ebc913">